modules/understanding-upgrade-channels.adoc

```diff
@@ -25,6 +25,8 @@ binary file for a specific version of {product-title} always installs that versi
 * `fast-{product-version}`
 * `stable-{product-version}`
 
+You can also configure an empty channel to tell the Cluster Version Operator to not fetch available updates from the upgrade recommendation service (for example, because your cluster has restricted network access and there is no local, reachable upgrade recommendation service).
```

Comment:
We should add the command to set empty channel here.

Reply:
I'd rather link to (existing?) docs about managing the channel via `oc` and the web console.  Can we address that in follow-up work?

Reply:
As long as the upgrade docs for the relevant scenarios have the command, I think that it's fine to say that you can do it here without the link or the command itself.

I'm not sure this statement makes sense here, though. You haven't introduced the upgrade service yet. Do we expect users to run this command outside of a restricted use case? If not, I'd put it under the *Restricted network clusters* heading.

If it stays here, I might say something like:
> You can also configure an empty upgrade channel, which prevents the cluster's Cluster Version Operator from fetching available upgrades from the upgrade recommendation service. Configure an empty upgrade channel only if the cluster cannot reach the upgrade recommendation services, such as when you have restricted network access.

If you move it, I'd say something like:
> In a cluster with restricted network access, the Cluster Version Operator cannot reach the upgrade recommendation service to fetch upgrades. You can configure an empty upgrade channel to prevent it from attempting to fetch upgrades.

Reply:
Rebased on master and picked up your wording for the first sentence with 635018ce5 -> eb93d29ca.  I kept my parenthetical instead of your "only if the cluster cannot reach" sentence, because I don't have a problem with folks opting out of the recommendation service in connected clusters.  I'd be fine expanding on the pros and cons of self-managing updates vs. using a recommendation service if we want to get into that here, but I'm also fine leaving both options on the table without much discussion (which would be distracting noise for folks who want update recommendations) and letting users sort that out on their own.

Reply:
@wking, the parenthetical sentence is half the reason I rewrote it - here's what our style guide says about them:

> Avoid using parentheses in running text because text in parentheses is decreased in importance.

The sentence is also a bit long as-is. Would this work?

> You can also configure an empty upgrade channel, which prevents the cluster's Cluster Version Operator from fetching available upgrades from the upgrade recommendation service. For example, you might configure an empty upgrade channel because your cluster has restricted network access and cannot reach the upgrade recommendation services.

Is opting out of the recommendation service in a connected cluster something that would give the users a good experience in production? If so, I think the future work might also need to include adding a section that provides that explanation later in the file.

Reply:
> Would this work...

Done with eb93d29ca -> e8552a573.

> Is opting out of the recommendation service in a connected cluster something that would give the users a good experience in production?

Maybe?  I could see folks disabling this if they had their own tooling that always called `oc adm upgrade --allow-explicit-upgrade --to-image ...` (like we do in CI, where we also have channels cleared since openshift/release#8631).  It's going to have no effect outside of whoever is managing cluster updates, and they're presumably the ones setting the channel property.  I'm fine allowing them to make whatever choice makes sense to them without putting an official thumb on the scale.